docs/development/API/Dshot.md

```diff
@@ -1,33 +1,35 @@
 
 - **Error checking**: a checksum allows the ESC to confirm that the data is truly what has been sent by the flight controller and there was no interference (at least to a certain degree)
-- **Higher resolution**: in case of DHSOT 2000 steps of throttle resolution
+- **Higher resolution**: in case of DShot, 2000 steps of throttle resolution
 - **No oscillator drift** and thus no need for calibration
-- **Two way communication on one wire**
+- **Two-way communication on one wire**
 
-But, everything has two sides, and so do digital protocols. The downsides are, that the digital protocols are not the fastest since they carry an overhead like - in the case of Dshot - the CRC, which adds reliability but also increases the duration of a frame and thus data needed to be transmitted. Also frames are always of a fixed length, no matter if you are going full throttle or no throttle - wheres the length of a pulse is shorter with analog when the throttle value is smaller.
+But, everything has two sides, and so do digital protocols. The downsides are, that the digital protocols are not the fastest since they carry an overhead like - in the case of DShot - the CRC, which adds reliability but also increases the duration of a frame and thus data needed to be transmitted. Also frames are always of a fixed length, no matter if you are going full throttle or no throttle - whereas the length of a pulse is shorter with analog when the throttle value is smaller.
 
-> Multishot has a maximum frame duration of 25µs at full throttle and is still more than twice as fast as Dshot 300 with a constant frame duration of 53.28µs.
+> MultiShot has a maximum frame duration of 25µs at full throttle and is still more than twice as fast as DShot 300 with a constant frame duration of 53.28µs.  
+  
+<br/>
 
 ## Supported Hardware
 
-Dshot is supported on all BLHELI_S, BLEHLI_32 and KISS ESC’s. One limitation are older BLHELI_S ESC’s with EFM8BB1 MCU: Only Dshot 150 and Dshot 300 are supported on those, but this should still be good enough for 99% of use cases.
+DShot is supported on all BLHELI_S, BLEHLI_32 and KISS ESC’s. One limitation are older BLHELI_S ESC’s with EFM8BB1 MCU: Only DShot 150 and DShot 300 are supported on those, but this should still be good enough for 99% of use cases.
 
-No extra settings on the ESC’s are needed - they automatically detect by which protocol they are driven and act accordingly. Some firmware might only support a certain subset of protocols though, so be aware of that. Bluejay for example only supports Dshot in all its variations, but none of the analog protocols.
+No extra settings on the ESC’s are needed - they automatically detect by which protocol they are driven and act accordingly. Some firmware might only support a certain subset of protocols though, so be aware of that. Bluejay for example only supports DShot in all its variations, but none of the analog protocols.
 
```

Comment:
Maybe “they automatically identify the incoming data protocol and act accordingly.” ?

Reply:
I agree, I'll make this change in a new PR.